test-requirements.txt

```diff
@@ -0,0 +1 @@
+cssselect
```

Comment:
Why is this needed?

Reply:
It is a dependency of the `resource_booking` addon that is only used in tests: https://github.com/OCA/calendar/blob/13.0/test-requirements.txt